docs/products/access/release-notes/1.x.md

```diff
@@ -1,5 +1,31 @@
 # Release notes 1.x versions
 
+## 1.1.1
+
+### Bug fixes
+
+* Resolved an issue causing some of the Access Admin's templates from being rendered correctly.
+
+## 1.1.0
+
+### Features
+
+* Mobile app developers can now modify the redirect URL that is configured in the Access Admin without breaking backwards compatibility with existing app installations.
```

Comment:
The mobile app developers don't have access to the admin console. 

WDYT of:
"It is now possible to configure multiple redirect URLs for mobile apps. This makes it possible to change the app scheme of the mobile app in a new version while the existing app installations use the old app scheme."

Reply:
Yes, you are right, such a message could be confusing. 
I like your proposal, updated 👍